Lib/trace.py

```diff
@@ -286,7 +286,7 @@ def write_results(self, show_missing=True, summary=False, coverdir=None, *,
             print("lines   cov%   module   (path)")
             for m in sorted(sums):
                 n_lines, percent, modulename, filename = sums[m]
-                print("%5d   %3d%%   %s   (%s)" % sums[m])
+                print(f"{n_lines:5d}   {percent/100:.1%}   {modulename}   ({filename})")
```

Comment:
This is incorrect.  That was suggestion:
```suggestion
                print(f"{n_lines:5d}   {n_hits/n_lines:.1%}   {modulename}   ({filename})")
```
You also should change above line to:
```
n_lines, n_hits, modulename, filename = sums[m]
```
and L282-283 to:
```
sums[modulename] = n_lines, n_hits, modulename, filename
```

Reply:
Oh, I seem to have misunderstood what you mean. I understand. I'll change to it.